.github/CODEOWNERS

```diff
@@ -0,0 +1,59 @@
+*                                  @coot  @karknu @nfrisby @EncodePanda @input-output-hk/consensus-superowners @input-output-hk/network-superowners
+
+# Network Team
+
+docs                               @coot @karknu @njd42 @peterwthomson @input-output-hk/network-superowners
+
+io-sim                             @coot @input-output-hk/network-superowners
+io-sim-classes                     @coot @input-output-hk/network-superowners
+
+network-mux                        @coot @karknu @input-output-hk/network-superowners
+ntp-client                         @coot @input-output-hk/network-superowners
+typed-protocols                    @coot @input-output-hk/network-superowners
+typed-protocols-examples           @coot @input-output-hk/network-superowners
+Win32-network                      @coot @input-output-hk/network-superowners
+
+ouroboros-network-framework        @coot @karknu @input-output-hk/network-superowners
+ouroboros-network                  @coot @karknu @input-output-hk/network-superowners
+ouroboros-network/wireshark-plugin @coot @karknu @input-output-hk/network-superowners
+ouroboros-network-testing          @coot @karknu @input-output-hk/network-superowners
+
+cardano-client                     @coot @input-output-hk/network-superowners
+
+scripts/haddocs.sh                 @coot @input-output-hk/network-superowners
+scripts/install.ps1                @coot @input-output-hk/network-superowners
+scripts/test.sh                    @coot @input-output-hk/network-superowners
+
+# Consensus Team
+
+ouroboros-consensus/               @nfrisby @input-output-hk/consensus-superowners
+ouroboros-consensus-test/          @nfrisby @input-output-hk/consensus-superowners
+ouroboros-consensus-mock/          @nfrisby @input-output-hk/consensus-superowners
+ouroboros-consensus-mock-test/     @nfrisby @input-output-hk/consensus-superowners
+ouroboros-consensus-byron/         @nfrisby @input-output-hk/consensus-superowners
+ouroboros-consensus-byronspec/     @nfrisby @input-output-hk/consensus-superowners
+ouroboros-consensus-byron-test/    @nfrisby @input-output-hk/consensus-superowners
+ouroboros-consensus-shelley/       @nfrisby @input-output-hk/consensus-superowners
+ouroboros-consensus-shelley-test/  @nfrisby @input-output-hk/consensus-superowners
+ouroboros-consensus-cardano/       @nfrisby @input-output-hk/consensus-superowners
+ouroboros-consensus-cardano-test/  @nfrisby @input-output-hk/consensus-superowners
+
+scripts/buildkite/check-stylish.sh @nfrisby @input-output-hk/consensus-superowners
+
+# CICD
+
+bors.toml                          @input-output-hk/devops @input-output-hk/consensus-superowners
+nix/                               @input-output-hk/devops @input-output-hk/consensus-superowners
+*.nix                              @input-output-hk/devops @input-output-hk/consensus-superowners
+.github/workflows/                 @input-output-hk/devops @input-output-hk/consensus-superowners
```

Comment:
Add `network-superowners` to these four too?

Reply:
Maybe we could have just one secrete team for that? In `network-superowners` there's Duncan, Philip and I (only because I didn't know how to remove myself).  Is there anybody else in `consensus-superowners? Maybe Edsko.

Reply:
Yeah; I'm not in it anymore (so it apparently is possible to remove yourself) but Edsko is.

My intuition is that we should have both groups and just pay the duplication price on some of the shared lines in this file. That seems the most future-compatible.

Reply:
:+1: